pkgs/checks/test/extensions/map_test.dart

```diff
@@ -29,6 +29,65 @@ void main() {
   test('values', () {
     check(_testMap).values.contains(1);
   });
+  test('can be described failing compared to another large map', () {
```

Comment:
Looks like the same issue exists for lists so it would be good to also test that (although I think your fix will work for both).